src/TootleLib/d3doverdrawwindow.cpp

```diff
@@ -807,23 +894,74 @@ CheckIsect(int iClusterA, int iClusterB)
                             + Vector3(m_vClusterDiag[iClusterA][0] / 2.f * i,
                                       m_vClusterDiag[iClusterA][1] / 2.f * j,
                                       m_vClusterDiag[iClusterA][2] / 2.f * k);
+
+#ifdef _DX11_1_
+                //vTransA
+                auto _xmv = DirectX::XMVectorSet(v[0], v[1], v[2], 0);
```

Comment:
Try not to prefix variable names with "_" character as this naming convention should applies to system variable names.